doc/doc-site/docs/off-chain/dss/websocket.md

```diff
@@ -31,7 +31,6 @@ Channels include:
 - `fill_event`
 - `change_order_size_event`
 - `cancel_order_event`
-- `recognized_market_event`
```

Comment:
Why is this being removed? The channel is still there.

Reply:
I think it got removed during a merge with an out of date PR or something. Adding it back.